test/plugins/individual-collateral/lido/constants.ts

```diff
@@ -19,18 +20,21 @@ export const MAX_TRADE_VOL = bn(1000)
 export const FORK_BLOCK = 14916729
 
 // Base Addresses
-// TODO: Switch to `networkConfig`
-export const BASE_WSTETH = '0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452' // confirm this
+export const BASE_WSTETH = networkConfig['8453'].tokens.wstETH as string
 export const BASE_WSTETH_WHALE = '0xa6385c73961dd9c58db2ef0c4eb98ce4b60651e8'
 export const FORK_BLOCK_BASE = 10264000
 export const BASE_PRICE_FEEDS = {
-  stETH_ETH: '0xf586d0728a47229e747d824a939000Cf21dEF5A0',
-  ETH_USD: '0x71041dddad3595F9CEd3DcCFBe3D1F4b0a16Bb70',
-  wstETH_stETH: '0xB88BAc61a4Ca37C43a3725912B1f472c9A5bc061',
+  // traditional finance notation, opposite of our unit system
+  stETH_ETH: '0xf586d0728a47229e747d824a939000Cf21dEF5A0', // {ETH/stETH}
```

Comment:
for consistency, lets put the addresses in the configuration.ts file and pull them into this file like we do for the `BASE_WSTETH` variable above